lightkurve/lightcurvefile.py

```diff
@@ -397,6 +405,20 @@ def __repr__(self):
         return('TessLightCurveFile(TICID: {})'.format(self.targetid))
 
     def get_lightcurve(self, flux_type, centroid_type='MOM_CENTR'):
+        if centroid_type+"1" in self.hdu[1].data.columns.names:
+            centroid_col = self.hdu[1].data[centroid_type + "1"][self.quality_mask]
+            centroid_row = self.hdu[1].data[centroid_type + "2"][self.quality_mask]
+        else:
+            centroid_col = self.hdu[1].data["TIME"][self.quality_mask]*0.0
+            centroid_row = self.hdu[1].data["TIME"][self.quality_mask]*0.0
```

Comment:
We'll want to use `np.zeros()` here too (or `np.zeros() * np.nan`?  we probably want nans here?)